cmd/oras/root/login.go

```diff
@@ -77,25 +80,34 @@ Example - Log in with username and password in an interactive terminal and no TL
 func runLogin(ctx context.Context, opts loginOptions) (err error) {
 	ctx, _ = opts.WithContext(ctx)
 
-	// prompt for credential
-	if opts.Password == "" {
+	// Check if both '--username' and '--identity-token' are provided
+	if opts.Username != "" && opts.IdentityToken != "" {
+		return fmt.Errorf("both --username and --identity-token cannot be used together")
+	}
+
+	// If IdentityToken is provided, use it as the credential without a username
+	if opts.IdentityToken != "" {
+		opts.Username = ""                 // Set the username to empty since it's not required when using identity token
+		opts.Password = opts.IdentityToken // Use the identity token as the password
+	} else if opts.Password == "" {
```

Comment:
Those part of code should be reverted if we introduce `opts.Secret`.

Reply:
Oh Okay. I will revert them. Let me know should I go with Secrets and SecretFromStdin or identity-token-stdin and password.